tests/system/data_sources/test_hive.py

```diff
@@ -375,3 +391,36 @@ def test_custom_query_validation_core_types():
     df = run_test_from_cli_args(args)
     # With filter on failures the data frame should be empty
     assert len(df) == 0
+
+
+@mock.patch(
+    "data_validation.state_manager.StateManager.get_connection_config",
+    new=mock_get_connection_config,
+)
+def test_row_validation_many_columns():
+    """Hive dvt_many_cols row validation.
+    Using BigQuery target because Hive queries are really slow.
+    When executed individually this test passes but when executed during a full
+    integration test run I get:
+      impala.error.HiveServer2Error: java.lang.NullPointerException
+    This must be down to minimal resources for our small Hive instance, disabling for now.
+    """
+    pytest.skip("Skipping test_row_validation_many_columns due to resource issues.")
```

Comment:
I suggest adding a TODO comment for us to remember to remove this skip eventually

Reply:
Agreed